src/amo/components/AddonReview/index.js

```diff
@@ -164,30 +164,43 @@ export class AddonReviewBase extends React.Component<Props, State> {
     }
 
     let placeholder;
-    let prompt;
+    let promptText;
     if (review.rating && review.rating > 3) {
-      prompt = i18n.gettext(
-        'Tell the world why you think this extension is fantastic!'
+      promptText = i18n.gettext(
+        `Tell the world why you think this extension is fantastic!
+        Please follow our %(linkStart)sreview guidelines%(linkEnd)s.`
       );
       placeholder = i18n.gettext(
         'Tell us what you love about this extension. Be specific and concise.'
       );
     } else {
-      prompt = i18n.gettext('Tell the world about this extension.');
+      promptText = i18n.gettext(
+        `Tell the world about this extension.
+        Please follow our %(linkStart)sreview guidelines%(linkEnd)s.`
+      );
       placeholder = i18n.gettext(
         'Tell us about your experience with this extension. ' +
         'Be specific and concise.'
       );
     }
 
+    const prompt = i18n.sprintf(promptText, {
+      linkStart: '<a href="/review_guide">', linkEnd: '</a>',
```

Comment:
We could really use a JS API for what  our `Link` component does - since this will presumably be a server redir + render with no locale?

This also looks like a case where we should consider opening the guidelines in a new tab since it will replace the current action the user is doing. That is unless clicking back recreates the modal and if the back button works then this can be ignored.

Bear in mind DOMPurify will remove target attrs by default and if you add target="_blank" you'll need rel="noopener noreferrer" to not leak the window object (albeit this is way less of an issue linking to ourselves but probably good to be consistent in that pattern in case it got copied for an external link later).

Reply:
We can always file any relevant issues as follow-ups to keep moving forward.

Reply:
I considered target=blank but I could not figure out how to get `sanitizeHTML()` to allow that. Is there a trick?

Reply:
I dont think it is feasible because of `dompurify`